_data/doc-categories.yml

```diff
@@ -78,7 +78,8 @@ extensibility:
     - extending-wiremock
     - extensibility/filtering-requests
     - extensibility/transforming-responses
-    - extensibility/custom-matching
+    - extensibility/custom-request-matching
+    - extensibility/custom-content-matching
```

Comment:
If implemented as is, it breaks the original page access. At least a redirect is needed in the markdown file